### ISSUE TITLE
Impossible to have empty prefix with JWT Bearer authentication

### ISSUE THREAD

**Issue by ahaverdings** [a85/POSTMan-Chrome-Extension#11841]
### Is there an existing issue for this?

- [X] I have searched the tracker for existing similar issues and I know that duplicates will be closed

### Describe the Issue

Authorization type "JWT Bearer" does not allow empty header prefix. Some servers (read: I'm dealing with older IBM Websphere servers here) do not expect the "Bearer " prefix to be present at all and will deny request when the authorization header has a prefix in the value but Postman will always default to "Bearer ".

**Setting:**
```
Algorithm=HS256
Secret={{Secret}}
Payload={ "hash": "1234567" }
Request header prefix=
JWT headers={ "type": "JWT" }
```
**Resulting header:**
`Authorization: Bearer eyJ0eXBlIjoiSldUIiwiYWxnIjoiSFMyNTYifQ.eyJoYXNoIjoiMTIzNDU2Nzg5In0.yIrvHYb4Q4QMHqhx-_ndLxucAyswoIr4WP9QCJoXCyA`
**Expected/required header**
`Authorization: eyJ0eXBlIjoiSldUIiwiYWxnIjoiSFMyNTYifQ.eyJoYXNoIjoiMTIzNDU2Nzg5In0.yIrvHYb4Q4QMHqhx-_ndLxucAyswoIr4WP9QCJoXCyA`

**Setting** (with space as prefix)
```
Algorithm=HS256
Secret={{Secret}}
Payload={ "hash": "1234567" }
Request header prefix=
JWT headers={ "type": "JWT" }
```
**Resulting header** (notice the extra spaces in the header value, this causes server to deny request as it includes the space as being part of token):
`Authorization:   eyJ0eXBlIjoiSldUIiwiYWxnIjoiSFMyNTYifQ.eyJoYXNoIjoiMTIzNDU2Nzg5In0.yIrvHYb4Q4QMHqhx-_ndLxucAyswoIr4WP9QCJoXCyA`

**Setting** (pointing to variable with empty value)
```
Algorithm=HS256
Secret={{Secret}}
Payload={ "hash": "1234567" }
Request header prefix={{emtpy_variable}}
JWT headers={ "type": "JWT" }
```
**Resulting header** :
`Authorization: Bearer eyJ0eXBlIjoiSldUIiwiYWxnIjoiSFMyNTYifQ.eyJoYXNoIjoiMTIzNDU2Nzg5In0.yIrvHYb4Q4QMHqhx-_ndLxucAyswoIr4WP9QCJoXCyA`

**Workaround**
Go to console and copy the authorization header (and removing the prefix) into Header tab for next request. This works but its not ideal as this requires at least one failed request and a manual step for each request after that.

### Steps To Reproduce

1. Open (new) request
2. Go to authorization tab and set authorization to "JWT Bearer"
3. Enter an empty value for "Request header prefix"
4. Execute request and inspect request "Authorization" header

### Operating System

macOS

### Postman Version

10.12.1

### Postman Platform

Postman App

### User Account Type

Signed In User

**Comment by giridharvc7** [comment]
Thanks for reporting this @ahaverdings , we'll keep you posted on this.

**Comment by giridharvc7** [comment]
Thank you for your patience, this is now fixed in v10.13. 
Please give this a try and let us know what you think

**Comment by ahaverdings** [comment]
Awesome! Confirmed its working 👍
